first_ruby/.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.7.3
```

Comment:
You said you use asdf. need this file?

Reply:
Removed this file & added `.tool-versions` instead